README.md

```diff
@@ -93,4 +87,18 @@ curl -X POST -H "Content-Type:application/json" --data-binary @segmentation/exam
     http://localhost:9876/jobs
 
 curl -H "Content-Type:application/json" -i http://localhost:9876/jobs/<job_id>
+```
+
+## Evaluating classification model running in container
+```
+pip install -r requirements.txt
+python classification/evaluate.py --api-url http://localhost:9876 \
+    --dataset-path classification/example_data/test_dataset --metric accuracy
+```
+
+## Evaluating segmentation model running in container
+```
+pip install -r requirements.txt
+python segmentation/evaluate.py --api-url http://localhost:9876 \
+    --dataset-path segmentation/example_data/test_dataset --class-n 3
```

Comment:
Launching the Docker container fails for me with the following error message: 
```
tivadar@lumina:~$ docker run -p 9876:9876 --name model-api --rm telestoai/model-api-example
docker: Error response from daemon: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"segmentation\": executable file not found in $PATH": unknown.
```

Reply:
Looks like you have an outdated base image, how have you built the model Docker image?
If you used the command from this README `docker build --pull -t telestoai/model-api classification/model`, it pulled an old base image. For development, you need to build an up to date base image yourself. `docker build -t telestoai/model-api-base -f Dockerfile .` in the telesto-base directory.

Reply:
This was the problem, it works now.